data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -256,9 +256,9 @@ until podman run \
 		--entrypoint etcdctl \
 		"${MACHINE_CONFIG_ETCD_IMAGE}" \
 		--dial-timeout=10m \
-		--cacert=/opt/openshift/tls/etcd-client-ca.crt \
-		--cert=/opt/openshift/tls/etcd-client.crt \
-		--key=/opt/openshift/tls/etcd-client.key \
+		--cacert=/opt/openshift/tls/etcd-ca-bundle.crt \
+		--cert=/opt/openshift/tls/etcd-signer-client.crt \
```

Comment:
after this change https://github.com/openshift/installer/pull/1720/files#diff-f19218c0ff3379e795ae4c5a125d73c9R97
`etcd-client.{crt,key}` are the correct file names.

Reply:
ok right they are the same actually but no need to change.

Reply:
well no, not now ack..

Reply:
fixed